features/support/env.rb

```diff
@@ -146,6 +146,12 @@
   Rails.env = 'test'
 end
 
+#Turn off logging output
```

Comment:
rather than a comment, should we make this a method name? e.g. 

```rb
def logger_with_no_output
  logger = double('Logger').as_null_object
  allow(Logger).to receive(:new).and_return(logger)
  logger
end

Before do
  logger = logger_with_no_output
end
```

and then it could even be dried out across the two sets of configs (although that's probably a dependency we don't want ...)